### PR TITLE
Timeline Replies Again

### PR DESCRIPTION
**Pull request by pdb493**
Now Tayne I can get into.

### CODE REVIEW

File: app/Http/Kernel.php [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 namespace App\Http;
 
+// TEST COMMENT AGAIN
```
[the remark this answers is not in the document]
Comment (pdb493) [reply app/Http/Kernel.php:5]: reply as customer

**Comment by pdb493** [comment]
> first summary comment submitted as reviewer
> 
> Image of Chanse T **Chanse T**
> 
> Was this helpful? [Yes](https://patrick-9a73.pullrequest.tech/reviews/ratings/4ece83d4-0ed7-46ed-8ab5-3715fb224823/true) | [No](https://patrick-9a73.pullrequest.tech/reviews/ratings/4ece83d4-0ed7-46ed-8ab5-3715fb224823/false)
> 
> > _Reviewers will be notified any time you reply to their comments or commit new changes._
> > _You can also request a full follow-up review from your [PullRequest dashboard](https://patrick-9a73.pullrequest.tech/dash/projects/1/pdb493/test/reviews/)._

quote reply as customer

File: app/User.php [hunk 2]
```diff
@@ -23,13 +24,7 @@ class User extends Authenticatable
...
      */
     protected $hidden = [
-        'password', 'remember_token',
+        'password', 
+	'remember_tokenz',
     ];
 }
-
-
-
```
Comment (pbarsallo) [inline app/User.php:31]: asdf

File: app/User.php [hunk 3]
```diff
@@ -23,13 +24,7 @@ class User extends Authenticatable
...
-
-
```
Comment (pbarsallo) [inline app/User.php:33]: asdf with review

**Pull request closed without merging**